From 47b47b7b3a807cc5ae3922ef5b67feb2c4f33e4f Mon Sep 17 00:00:00 2001
From: "Frederic Pierret (Epitre)" <frederic.epitre@orange.fr>
Date: Tue, 19 Sep 2017 18:33:57 +0200
Subject: [PATCH 14/31] anaconda: drop useless on Qubes dependencies on network
 filesystems

Installing Qubes on network drive is not supported, so drop those dependencies.
---
 pyanaconda/kickstart.py | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/pyanaconda/kickstart.py b/pyanaconda/kickstart.py
index 57ba413..ccc7f15 100644
--- a/pyanaconda/kickstart.py
+++ b/pyanaconda/kickstart.py
@@ -2301,12 +2301,6 @@ def parseKickstart(f, strict_mode=False):
 
     # We need this so all the /dev/disk/* stuff is set up before parsing.
     udev.trigger(subsystem="block", action="change")
-    # So that drives onlined by these can be used in the ks file
-    blivet.iscsi.iscsi.startup()
-    blivet.fcoe.fcoe.startup()
-    blivet.zfcp.zfcp.startup()
-    # Note we do NOT call dasd.startup() here, that does not online drives, but
-    # only checks if they need formatting, which requires zerombr to be known
 
     kswarnings = []
     ksmodule = "pykickstart"
-- 
2.9.5

